x-pack/plugins/spaces/server/lib/request_inteceptors/on_post_auth_interceptor.ts

```diff
@@ -80,8 +80,8 @@ export function initSpacesOnPostAuthRequestInterceptor(server: KbnServer) {
       }
 
       // Verify application is available in this space
-      if (space && space.disabledFeatures.length > 0) {
-        const appId = path.split('/', 3)[2];
+      const appId = path.split('/', 3)[2];
```

Comment:
The management "application" is always visible, so we shouldn't be restricting access to the `kibana` application in any situation.

Reply:
Great call, will do.